src/mono/mono/mini/CMakeLists.txt

```diff
@@ -381,6 +381,42 @@ if(NOT DISABLE_SHARED_LIBS)
     add_library(monosgen-shared-dac SHARED "mini-windows-dlldac.c")
     set_target_properties(monosgen-shared-dac PROPERTIES OUTPUT_NAME ${MONO_SHARED_LIB_NAME}-dac)
   endif()
+
+  if(BUILD_DARWIN_FRAMEWORKS)
+    if(TARGET_DARWIN)
+      string(REPLACE ";" "*" mono-components-objects-nowhitespace "${mono-components-objects}")
+      string(REPLACE ";" "*" mono-components-stub-objects-nowhitespace "${mono-components-stub-objects}")
+      list(APPEND FrameworkConfig Mono.release Mono.debug)
+      list(APPEND ComponentsObjects "${mono-components-objects-nowhitespace}" "${mono-components-stub-objects-nowhitespace}")
+      foreach(frameworkconfig componentsobjects IN ZIP_LISTS FrameworkConfig ComponentsObjects)
```

Comment:
Isn't this backwards? the release framework gets the full components, and the debug framework gets the stubs?